packages/documentation/copy/en/handbook-v2/Narrowing.md

```diff
@@ -111,8 +111,6 @@ This might be a good segue into what we'll call "truthiness" checking.
 
 Truthiness might not be a word you'll find in the dictionary, but it's very much something you'll hear about in JavaScript.
 
-<!-- TODO: I'm on an airplane, is truthiness in the dictionary?? -->
```

Comment:
it was

![Screen Shot 2020-10-08 at 7 44 32 PM](https://user-images.githubusercontent.com/49038/95524064-56c27780-099e-11eb-8510-83718192d73c.png)